cluster-scope/overlays/prod/moc/smaug/snapshotschedules/schedule.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: snapscheduler.backube/v1
+kind: SnapshotSchedule
+metadata:
+    name: daily
+    namespace: openshift-storage
+spec:
+    claimSelector:
+        matchLabels:
+            noobaa-db: postgres
+    retention:
+        expires: “168h”
+    schedule: “0 0 * * *”
```

Comment:
Note that despite appearances, this will not run at midnight unless you happen to live in the UK. From [the documentation](https://backube.github.io/snapscheduler/usage.html):

> Dates and times specified in the cronspec are relative to the UTC timezone (i.e., a schedule of "0 5 * * *" will create a snapshot once per day at 5:00 AM UTC).

I don't think we need to change it, but it might worth a comment if you're changing something else (`# Note that schedules are in the UTC timezone`).

Reply:
Totally missed the timezone detail! I'll add in a comment.